rentoshare-jdl.jdl

```diff
@@ -0,0 +1,137 @@
+
+application {
+  config {
+    baseName gateway // (1)
```

Comment:
why the comments and the numbers?

Reply:
If you don't like anything on it , I'm sure you can change it yourself.